api/saml/metadata/federations/incommon.py

```diff
@@ -0,0 +1,33 @@
+FEDERATION_TYPE = "InCommon"
+
+IDP_METADATA_SERVICE_URL = "https://mdq.incommon.org/entities/idps/all"
+
+CERTIFICATE = """-----BEGIN CERTIFICATE-----
```

Comment:
Ideally a certificate should not be in the code base but instead stored somewhere in either a configuration file or database. This is ok for now but we'll want to move things like certs & secret keys to configuration in the future.

Reply:
This does get put into the database as part of a migration script. This also tripped me up--putting this data in  its own module makes it look like it's going to be imported and used as a source of constants, rather than inserted into the database and pulled out later.

I don't see a way to make this stuff end up in the database upon initialization without including it in the source code, but it does need to be revisited so it doesn't rely on a specific migration script having run.